python/src/opendp/context.py

```diff
@@ -378,6 +378,24 @@ def __repr__(self) -> str:
     d_mids     = {self.d_mids})"""
     # TODO: Add "d_out" when filters are implemented.
 
+    def deserialize_polars_plan(self, serialized_plan: bytes) -> "LazyFrameQuery":
+        '''
+        Given a serialized Polars plan, wraps it with a LazyFrameQuery.
+        See the :ref:`serialization documentation <lazyframe-serialization>`
+        for context and full example.
+
+        :param serialized_plan: A plan like that returned by ``query.polars_plan.serialize()``
+        '''
+        import io
+        pl = import_optional_dependency('polars')
+        
+        new_plan = pl.LazyFrame.deserialize(io.BytesIO(serialized_plan))
+        new_query = self.query()
+        assert isinstance(new_query, LazyFrameQuery) # Lets us tighten the output type.
```

Comment:
Irregardless of typing, I think it would be better to give an error that says the data must be a lazyframe to use this method.

```suggestion
	if not isinstance(new_query, LazyFrameQuery):
		raise ValueError("deserialize_polars_plan may only be used when data is a LazyFrame")
```

Reply:
For the case where a user creates a normal Context, but calls this method and would get an assertion error without explanation.